addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6728,7 +6766,43 @@ registry.ImageTools = ImageHandlerOption.extend({
      * @see this.selectClass for parameters
      */
     async setImgShape(previewMode, widgetValue, params) {
+        this.trigger_up("disable_loading_effect");
         const img = this._getImg();
+        const restoreCursor = preserveCursor(this.$target[0].ownerDocument);
+        const document = this.el.ownerDocument;
+        const isCropRequired = this._isCropRequired();
```

Comment:
I think it is more correct to use `params.unstretch` (even if they give the same value) :slightly_smiling_face: